addons/test_mail_full/tests/test_portal.py

```diff
@@ -75,6 +75,17 @@ def test_redirect_to_records_norecord(self):
                 'Failed with %s - %s' % (model, res_id)
             )
 
+    def test_portal_avatar_image(self):
+        # print(self.record_portal)
+        # partner_portal = self._create_portal_user()
+        mail_record = self.env['mail.message'].create({
+            'author_id': self.record_portal.partner_id.id,
+            'model': self.record_portal._name,
+        })
+        response = self.url_open(f'/mail/avatar/mail.message/{mail_record.id}/author_avatar/50x50?access_token={self.record_portal.access_token}')
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.headers.get('Content-Type'), 'image/png')
+
```

Comment:
Both the placeholder and the author avatar are status_code=200 and Content-Type: image/png. Please add an assertion on the Content-Disposition header or the response content to disambiguate the two.